deploy/upstream/05_role.yaml

```diff
@@ -76,3 +88,11 @@ rules:
   - update
   - list
   - watch
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - servicemonitors
+  verbs:
+  - get
+  - create
+  
```

Comment:
Something is off here.

Reply:
This was a mistake. Reverted to the old version and made the required changes